src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -920,8 +920,8 @@ def describe_service(self, service_type=None, service_name=None, refresh=False):
         """
         raise NotImplementedError()
 
-    def list_daemons(self, daemon_type=None, daemon_id=None, host=None, refresh=False):
-        # type: (Optional[str], Optional[str], Optional[str], bool) -> Completion
+    def list_daemons(self, service_name=None, daemon_type=None, daemon_id=None, host=None, refresh=False):
```

Comment:
for the sake of consistency:

```suggestion
    def list_daemons(self, service_type=None, service_name=None, daemon_id=None, host=None, refresh=False):
```

Reply:
@sebastian-philipp 
During my refactoring, I feel that keeping `daemon_type` is more natural, since `list_daemons` call mainly operates on `DaemonDescription` and `daemon_type` is an attribute of if.
(This also introduces lesser code change).

How about keeping `daemon_type`?

Reply:
Think we need a large-scale refactoring to unify `service_type` and `daemon_type` throughout  cephadm

Reply:
Agree. Just rebased, but still keeping daemon_type.